userspace/libsinsp/threadinfo.cpp

```diff
@@ -360,6 +360,7 @@ void sinsp_threadinfo::init(scap_threadinfo* pi)
 
 	m_comm = pi->comm;
 	m_exe = pi->exe;
+	m_exepath = pi->exepath;
```

Comment:
If I'm reading this correctly, this part of the code could be called when you're reading the initial state of a process serialized in a trace file. However, it doesn't seem that `scap_read_proclist` correctly initializes `exepath`, so this could effectively be garbage?